### ISSUE TITLE
Space remains after clearing checks

### ISSUE THREAD

**Issue by Eagle-Man** [al-caughey/Google-Meet-Attendance#11]
I believe when you add a checkmark or other symbol before a student's name, you also enter a space character, but when clearing, that space character remains when it should probably be deleted.  It's a small detail, but I noticed it when copying my existing lists from the extension into a text file before a Reset.

**Comment by AlCaughey** [comment]
Yes, now that you mention it, I know exactly what is happening.  It'll be
easy to fix

Thanks for your continued feedback

**Comment by al-caughey** [comment]
fixed... please confirm and re-open if necessary

**Comment by Eagle-Man** [comment]
Confirmed.  Thanks!
